packages/app-builder-lib/src/options/PlatformSpecificBuildOptions.ts

```diff
@@ -50,6 +50,13 @@ export interface PlatformSpecificBuildOptions extends TargetSpecificOptions {
    */
   readonly compression?: CompressionLevel | null
 
+  /**
+   * Whether to ignore all default files(https://www.electron.build/configuration/contents#files) and options. Defaults to `false`.
+   *
+   * @default false
+   */
+  ignoreDefaultFiles?: boolean
```

Comment:
Minor: I think we may want to change the name of this variable, such `excludeDefaultFiles` (default true) or `disableDefaultIgnoredFiles` (default false. Might be too long of a var name though)

Reply:
+1 for `disableDefaultIgnoredFiles` very clear and you write this field once as part of a config.

Reply:
Fixed